src/main/server/databases/transformers/AttributedBodyTransformer.ts

```diff
@@ -0,0 +1,7 @@
+import { Server } from "@server/index";
+import { ValueTransformer } from "typeorm";
+
+export const AttributedBodyTransformer: ValueTransformer = {
+    from: dbValue => Server().swiftHelper.deserializeAttributedBody(dbValue),
```

Comment:
If the `dbValue` is null, will this error out?

Reply:
It won't error out, however it will still send it to the swift helper (which probably isn't ideal), I'll add a null check in a bit